src/process/pidfd_getfd.rs

```diff
@@ -0,0 +1,42 @@
+#![allow(unsafe_code)]
+use crate::fd::OwnedFd;
+use crate::{backend, io};
+use backend::fd::{AsFd, RawFd};
+
+/// Raw file descriptor in another process.
```

Comment:
It'd be good to add a note here that users should take extra care to avoid using host file descriptors here, since they use the same underlying type.